package.json

```diff
@@ -2,25 +2,31 @@
   "name": "@ngneat/transloco-keys-manager",
   "version": "3.8.0",
   "description": "Extract translatable keys from projects that uses Transloco",
-  "main": "public_api.js",
+  "engines": {
+    "node": ">=18.13.0"
+  },
+  "type": "module",
+  "exports": {
+    "types": "./public-api.d.ts",
+    "default": "./public-api.js"
+  },
+  "bin": {
+    "transloco-keys-manager": "index.js"
+  },
   "scripts": {
-    "postinstall": "npx husky install",
     "release": "standard-version --infile ./CHANGELOG.md",
     "commit": "git-cz",
-    "test": "cross-env PRODUCTION=true jest",
+    "test": "cross-env PRODUCTION=true node --no-warnings --experimental-vm-modules node_modules/jest/bin/jest.js",
     "start": "npm run prettier && tsc --watch",
     "prebuild": "npm run clean:dist",
-    "build": "tsc",
+    "build": "tsc && tsc-alias",
```

Comment:
Is the only usage for `tsc-alias` is for appending the `js` file extension?

Reply:
Yes, I discovered the extent of this thing and the endless debate between Typescript and the community about whether it should be automatic.
I clearly didn't want to do it by hand and found this utils.

Reply:
Yes manually is not an option.
I just find it surprising that it's not resolved within Typescript itself.
Tnx, I'll maybe look into in the future